### ISSUE TITLE
Enhancement: Attributes inheritance in Files and DirFiles for generated items (File and Dir)

### ISSUE THREAD

**Issue by manytwo** [oleg-shilo/wixsharp#26]
Extend WixSharp.DirFiles and WixSharp.Files to support attributes inheritance for generated items.

## WixSharp.DirFiles
```CS
new DirFiles(feature1, @"Files\Feature1\Docs\*.*")
{
    AttributesDefinition = "Attribute1=value1;Attribute2=value2"
},
```
Should Generate collection of 
```CS
new File(feature1, @"Files\Feature1\Docs\File1")
{
    AttributesDefinition = "Attribute1=value1;Attribute2=value2"
},
...
```
## WixSharp.Files
Files is recursive and generate WixSharp.File and WixSharp.Dir item, so we need to differentiate target.
Should use same syntax as Component cibling. As Component can't have a Dir children,
"Component :Attribute=value" can target File element.
```CS
new Files(feature1, @"Files\Feature1\Docs\*")
{
    AttributesDefinition = "File:Attribute1=value1;Component:Attribute2=value2;Dir:Attribute3=value3"
},
```
Should Generate collections of 

```CS
new File(feature1, @"Files\Feature1\Docs\File1")
{
    AttributesDefinition = "Attribute1=value1;Component:Attribute2=value2"
},
...
```
```CS
new Dir(feature1, "MyDir")
{
    AttributesDefinition = "Attribute3=value3"
},
...
```

**Comment by oleg-shilo** [comment]
Thank you,
Just has been implemented in 178dccdd0016be00d6e7ad9622787ee2ccdb1441
Waiting for the next release to publish.

**Comment by manytwo** [comment]
Nice! Good job!
